### ISSUE TITLE
UrlPattern stringify options

### ISSUE THREAD

**Issue by davecoates** [prestojs/prestojs#61]
I've changed url generation to use the `indices: false` option so matches how django works:

```js
url = `${url}?${qs.stringify(query, { indices: false })}`;
```

but this depends on backend. How do we handle this generically?

**Comment by fanglong-alliance** [comment]
a generic configuration on runtime init(looks like the approach most took), an option on urlpattern (better support if need to talk w/ various endpoint providers), or have the backend telling front how to behave dynamically(embedded in our "3rd party" so users dont need to know)?

**Comment by davecoates** [comment]
I think probably both a runtime init option (similar to config supported on Endpoint) and option to UrlPattern (defaults to default config controlled by runtime init)

**Comment by levic** [comment]
Yeah, you definitely need both -- systems with multiple backend API aren't that uncommon (even if they weren't, I expect TT to be a big consumer of presto and that has multiple backends)
